core/display.js

```diff
@@ -88,7 +81,39 @@ try {
 }
 
 Display.prototype = {
-    // Public methods
+    // ===== PROPERTIES =====
+
+    _scale: 1.0,
+    get scale() { return this._scale; },
```

Comment:
this is ES6 syntax.  Do all of our intended platforms support it?

Reply:
(P.S. I'm fine with using ES6 syntax if our intended platforms support it, or even saying that we should transpile for older platforms, but it's something we need to decide on.  If we do, we can probably just switch to class syntax...)

Reply:
MDN claims so. Haven't tested it yet.